### ISSUE TITLE
optionally allow the state holding object to specify the transition timeout

### ISSUE THREAD

**Issue by arlac77** [arlac77/statetransition-mixin#1]
add method like:

transitionTimeout(transition)

**Comment by arlac77** [comment]
```javascript
getTransition(action) {
   return Object.asign({},action.initial[this.state],{ timeout: 4711 });
}
```

**Comment by arlac77** [comment]
timeoutForTransition(transition)
fixed with #2c6d458b9d26fcad988b454869be98f46736c8d5
